salviosage/AutoMart
# #165947276  build products display UI page

**What does this PR do?**

- 
display all available products 

**Description of Task to be completed?**

- user can see available cars ads for sale  

How should this be manually tested?

- by clicking on products nav link you will see the products page or after clone this repo go to UI folder to pages folder you will see products.html and open it 

What are the relevant pivotal tracker stories?
#165947276
